### PR TITLE
Fixed `;` for TypeScript weapon enum and added the new DLC name to dlc.md

### PR DESCRIPTION
**Pull request by Phill030**

### CODE REVIEW

File: articles/references/dlcs.md [hunk 1]
```diff
@@ -220,6 +224,8 @@ public enum GtaDlc
 	mptuner,
 	mpsecurity,
 	mpg9ec,
-	patchdayg9ecng
+	patchdayg9ecngm
```
Comment (C0kkie) [inline articles/references/dlcs.md:227]: dot missing

File: articles/references/dlcs.md [hunk 2]
```diff
@@ -72,6 +72,8 @@ The release dates are using the format **YYYY.MM.DD** and PC release dates are p
 | mpg9ec | 2022.04.26 |
 | patchdayg9ecng |  |
 | mpg9ecpatch | 2022.05.17 |
+| mpsum2 | 2022.07.26 |
+| mpsum2_g9ec |
```
Comment (C0kkie) [inline articles/references/dlcs.md:76]: date missing

File: articles/weapons/models.md [hunk 3]
```diff
@@ -1151,115 +1151,115 @@ export class WeaponModel {
 # [TypeScript](#tab/tab1-1)
 ```ts
 export enum WeaponModel {
```
Comment (C0kkie) [inline articles/weapons/models.md:1153]: remove that file from your pr, that is autogenerated

Comment (Phill030) [reply articles/weapons/models.md]: Can't  remove the file from the PR in Web-Mode I can only delete the file

**Comment by C0kkie** [comment]
You deleted now the whole file xd you should just revert your changes

**Comment by Phill030** [comment]
I think I'm gonna close this and create one without the models.md

**Pull request closed without merging**